modules/insight/src/main/PovToEntry.scala

```diff
@@ -143,8 +137,8 @@ final private class PovToEntry(
 
         InsightMove(
           phase = Phase.of(from.division, ply),
-          tenths = movetime.roundTenths,
-          clockPercent = ClockPercent(from.clock, clock),
+          tenths = from.movetimes.flatMap(_.lift(i)).map(_.roundTenths),
+          clockPercent = from.clock.flatMap(clk => from.clockStates.flatMap(_.lift(i)).map(ClockPercent(clk, _))),
```

Comment:
Not sure if there's a better way to do this than using lift? 🤔

Reply:
`.zipAll(from.movetimes | Vector.fill(roles.size, none))`

Reply:
Hm, didn't quite work like that but I guess I got the idea